test/pageObjects/wizardPages/conditionals_CreateCase_ConditionalPage1.js

```diff
@@ -0,0 +1,49 @@
+let CCDStringField = require('../ccd-components/fields/ccdStringField.js');
+
+class ConditionalsCreateCasePage1WizardPage extends CreateCaseWizardPage {
+
+  constructor() {
+    super();
+    this._fieldUsedInShowCondition = ''; // TextField | TextFieldOptional
+    this.mandatoryTextField = new CCDStringField('ccd-write-text-field', 'text', 'TextField');
```

Comment:
Nothing wrong with this, but nothing stopping us just passing the ID in as the first param as either `#TextField` or `ccd-write-text-field #TextField` not sure why we have a id param when we already have a css param that an id can be parsed into. (Just some general thoughts)